src/ctim/domain/disposition.cljc

```diff
@@ -0,0 +1,18 @@
+(ns ctim.domain.disposition)
+
+(def ^:private weights
+  {"Clean"      1
+   "Malicious"  2
+   "Suspicious" 3
+   "Unknown"    4})
```

Comment:
I'm not 100% certain this has some importance, but I prefer to make the remark.

Regarding the disposition in verdict we also have "Common" with value 4 and "Unknown" for value 5:

~~~clojure
                  "clean" 1
                  "malicious" 2
                  "suspicious" 3
                  "common" 4
                  "unknown" 5
~~~